cmd/step/main_test.go

```diff
@@ -82,3 +85,47 @@ func TestStepTrainSQL(t *testing.T) {
 	INTO sqlflow_models.mytest_model;`
 	a.NoError(checkStepWrapper(func() { a.NotPanics(func() { runSQLStmt(sql, session) }) }, trainLogChecker))
 }
+
+func TestStepStandardSQL(t *testing.T) {
+	if os.Getenv("SQLFLOW_TEST_DB") != "mysql" {
+		t.Skip("skip no mysql test.")
+	}
+	a := assert.New(t)
+	dbConnStr := "mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0"
+	session := makeTestSession(dbConnStr)
+	sql := `SELECT * FROM iris.train limit 5;`
+	a.NoError(checkStepWrapper(func() { a.NotPanics(func() { runSQLStmt(sql, session) }) }, func(s string) error {
```

Comment:
Maybe just reuse `getStdout` in `repl_test.go`, passing two functions here is hard to read.